docs/pages/install/docker.mdx

```diff
@@ -69,7 +69,7 @@ To run the latest pelican registry server, run the following command:
 docker run -it -p 8444:8444 --name=pelican-registry hub.opensciencegrid.org/pelican_platform/registry:latest serve -f <federation>
 ```
 
-> Note that to successfully run a Pelican registry server, additional configuration is required. Follow [Serve a Pelican Federation](../serving_a_federation.mdx) for instructions.
+To successfully run a Pelican registry server, additional configuration is required. Follow [Serve a Pelican Registry](../serving_a_federation.mdx#serve-a-registry) for instructions. For how to pass configurations to Docker container, refer to [the next section](#configure-pelican-server-in-a-container).
```

Comment:
Should this statement go before you show the example command for running the director container? 
My concern is that people will see and try to run the command before reading the note that it won't work as it is.